### DISCUSSION TITLE
Slow menu and groups tree navigation

### DISCUSSION THREAD

**Discussion by TigBuh99** [keepassxreboot/keepassxc#9120] [Q&A] [answered]
I experience a delay when using the mouse to access the menus or the groups tree. I click on the menu or the groups tree and there is a long delay before it opens or there is a response. I'm using a database that I transferred from Keepass and I'm on Windows 11. Is there a setting that could fix this?

EDIT: Replying to me. It was fixed after I rebooted the PC. I don't know if the problem had anything to do with the fact that I had Keepass also open at the time, but I don't have time to test this possibility.

**Comment by droidmonkey** [comment, marked as answer]
We don't interact in any way with the keepass program, likely just your computer being silly.

**Reply by TigBuh99** [reply]
Thank you. I'm glad that it was all back to normal after a reboot.
